easybuild/easyconfigs/d/Dakota/Dakota-6.11.0-foss-2021b.eb

```diff
@@ -0,0 +1,63 @@
+# /!\ IMPORTANT: you need at least 8 cores to complete successfully the tests
+
+easyblock = "CMakeMake"
+
+name = "Dakota"
+version = "6.11.0"
```

Comment:
Do we have a reason to stick to 6.11.0 when 6.16.0 is already there?
https://dakota.sandia.gov/release-notes/6.16